content/rs/release-notes/rs-7-4-2-releases/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: Redis Enterprise Software release notes 7.4.2
+linkTitle: 7.4.2 releases
+description: New Cluster Manager UI enhancements, including Active-Active database management. TLS 1.3 support. Automatic recovery configuration. IPv6 support for internal traffic. Maintenance mode enhancements. Two feature sets.
+compatibleOSSVersion: Redis 7.2.0
+weight: 70
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+---
+
+​[​Redis Enterprise Software version 7.4.2](https://redis.com/redis-enterprise-software/download-center/software/) is now available!
+
+## Highlights
+
+This version offers:
+ 
+- New Cluster Manager UI enhancements, including Active-Active database management
+
+- TLS 1.3 support
```

Comment:
@rrelledge "TLS 1.3 support" suggests that we didn't have TLS 1.3 support at all until now, which is wrong and misleading. 
(It's my bad since I suggested the current phrasing...)
I believe that we should change this to something like: 
"TLS 1.3 full support"
Which suggests that it was partially supported until now, which is more accurate.

Reply:
Fixed